### DISCUSSION TITLE
[Android] Stream video full screen.

### DISCUSSION THREAD

**Discussion by duy-maimanh** [google/filament#4548] [Q&A] [answered]
Hello. I am trying sample-hello-camera. The camera is displayed on the front of the cube.
Now i want display it full screen. And I change the size of cube.

            .put(Vertex(-2.5f, -2.5f, -1.0f, tfNZ))
            .put(Vertex(-2.5f, 2.5f, -1.0f, tfNZ))
            .put(Vertex(2.5f, 2.5f, -1.0f, tfNZ))
            .put(Vertex(2.5f, -2.5f, -1.0f, tfNZ))
            // Face +X, right
            .put(Vertex(2.5f, -2.5f, -1.0f, tfPX))
            .put(Vertex(2.5f, 2.5f, -1.0f, tfPX))
            .put(Vertex(1.2f, 1.2f, 1.0f, tfPX))
            .put(Vertex(1.2f, -1.2f, 1.0f, tfPX))
            // Face +Z front
            .put(Vertex(-1.2f, -1.2f, 1.0f, tfPZ))
            .put(Vertex(1.2f, -1.2f, 1.0f, tfPZ))
            .put(Vertex(1.2f, 1.2f, 1.0f, tfPZ))
            .put(Vertex(-1.2f, 1.2f, 1.0f, tfPZ))
            // Face -X left
            .put(Vertex(-1.2f, -1.2f, 1.0f, tfNX))
            .put(Vertex(-1.2f, 1.2f, 1.0f, tfNX))
            .put(Vertex(-2.5f, 2.5f, -1.0f, tfNX))
            .put(Vertex(-2.5f, -2.5f, -1.0f, tfNX))
            // Face -Y bottom
            .put(Vertex(-1.2f, -1.2f, 1.0f, tfNY))
            .put(Vertex(-2.5f, -2.5f, -1.0f, tfNY))
            .put(Vertex(2.5f, -2.5f, -1.0f, tfNY))
            .put(Vertex(1.2f, -1.2f, 1.0f, tfNY))
            // Face +Y top
            .put(Vertex(-2.5f, 2.5f, -1.0f, tfPY))
            .put(Vertex(-1.2f, 1.2f, 1.0f, tfPY))
            .put(Vertex(1.2f, 1.2f, 1.0f, tfPY))
            .put(Vertex(2.5f, 2.5f, -1.0f, tfPY))

And I got an issue (You can see in the image below.)
![20210826_162642](https://user-images.githubusercontent.com/58172953/130947472-ca0d85ee-917a-4745-aafc-3cda10438308.jpg)

I guess the reason from the lit.filamat file. Do you have any suggestions?

**Comment by romainguy** [comment]
This sample uses the vertex positions as the UV coordinates:

```
material.uv = 0.5 * (getPosition() + vec4(1));
```

You'll have to change this since you want UVs between 0 and 1 to display the full image. You could also just set UV coordinates on your vertices and use that in the material.

**Reply by duy-maimanh** [reply]
Hi @romainguy .Thank you for your answer. 
After I set the UVs to 0.2 the picture was showing bigger. But it's only bigger to the right and to the top.
I tried Matrix.translateM but it doesn't seem to work. Did I forget something ?
![20210831_095431](https://user-images.githubusercontent.com/58172953/131435058-d583ea26-6cb1-455c-a8a3-549c2dd3b54b.jpg)

**Comment by romainguy** [comment, marked as answer]
You should just set a UV attribute on your vertices it would be much easier.
